data_processing/pathology/proxy_table/generate.py

```diff
@@ -140,8 +145,7 @@ def create_proxy_table(config_file):
 
     # parse all dicoms and save
     df.printSchema()
-    print (df.first().metadata)
-    df.show()
+    df.coalesce(48).write.format("delta").save(write_uri + wsi_path)
```

Comment:
it would be nice to have the num_partitions come from DATA_CFG also, instead of having it hard-coded.

Reply:
Noted, something we should optimize for sure.
 Learned that this choice is also quite complicated, and is a function of the row size in MB and target partition size.